PowerShell/JumpCloud Module/Docs/Set-JCPolicy.md

```diff
@@ -122,6 +124,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -registryFile
```

Comment:
Coming back to this, we did update this param to be dynamic in New-JCPolicy.ps1, we should do the same for Set-JCPolicy.ps1.